templates/blocks/blok_dossier.twig

```diff
@@ -1,38 +1,45 @@
-<div class="bg-gray-800 text-white">
+<div class="bg-gray-800 text-white my-4">
+    <div class="lg:hidden">
+        <div class="aspect-w-21 aspect-h-9">
+            <div>
+                <img
+                    class="object-cover absolute inset-0 w-full h-full"
+                    src="{{ Image(block.term.dossier_afbeelding_breed).src|thumbor(1024, 439) }}"
+                    srcset="{{ Image(block.term.dossier_afbeelding_breed).src|thumbor(1024*2, 439*2) }} 2x"
+                />
+                <div class="absolute inset-x-0 bottom-0 h-32 max-h-full bg-gradient-to-t from-gray-800"></div>
+            </div>
+        </div>
+    </div>
     <div class="w-full mx-auto max-w-960">
         <div class="lg:grid grid-cols-3">
             <div class="lg:col-span-2 lg:col-start-2 relative z-20">
                 <div class="px-4 pt-4">
-                    {% if block.tekst_boven_dossier %}
-                        <h1 class="text-3xl font-bold">{{ block.tekst_boven_dossier }}</h1>
-                    {% else %}
-                        <h1 class="font-bold">
-                            <span class="block text-sm tracking-wide uppercase font-round">Dossier</span>
+                    <h1 class="font-bold">
+                        <span class="rounded-md px-2 py-0.5 text-sm tracking-wide uppercase font-black font-round bg-blauw text-white">📂 Dossier</span>
```

Comment:
Andere tags zijn text-xs. Also, wit-op-gray is wat neutraler.
<img width="440" alt="IMG_0779" src="https://user-images.githubusercontent.com/737603/158081209-d8a94e0f-4a8a-4dc6-b99d-41d0096c09ea.png">